app/Http/Controllers/Admin/BookController.php

```diff
@@ -32,15 +34,11 @@ public function index()
     //FormRequest
     public function store(BookPostRequest $request)
     {
-        $filePath = false;
         $book = new Book();
-
-        if ($request->file("image") != null) {
-            $filePath = Storage::disk('uploads')->put('books', $request->file("image"), 'public');
-        }
-
-        if ($filePath == false) {
-            $filePath = Storage::disk('uploads')->put('books', $request->file("image"), 'public');
+        if ($request->hasFile('image')) {
+            $filePath = Storage::disk('storage')->put('books', $request->file("image"), 'public');
+        } else {
+            $filePath = "no-image/no-image.jpeg";
```

Comment:
daha önceki reviewde bahsettiğin early return mevzusu mesela burada da uygulanmalı.

```php
        if ($request->hasFile('image')) {
            $filePath = Storage::disk('storage')->put('books', $request->file("image"), 'public');
        } else {
            $filePath = "no-image/no-image.jpeg";
        }
```

early return hali:

```php
        $filePath = "no-image/no-image.jpeg";
        if ($request->hasFile('image')) {
            $filePath = Storage::disk('storage')->put('books', $request->file("image"), 'public');
        }
```